atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyUnicastService.java

```diff
@@ -141,7 +151,7 @@ protected void channelRead0(
                 })
             .option(ChannelOption.RCVBUF_ALLOCATOR, new DefaultMaxBytesRecvByteBufAllocator())
             .option(ChannelOption.SO_BROADCAST, true)
-            .option(ChannelOption.SO_REUSEADDR, true);
+            .resolver(dnsAddressResolverGroup);
```

Comment:
:x: If we manually call the resolver group, then I think here this is completely unused and can be removed. In fact, I'm pretty sure it's unused because here `dnsAddressResolverGroup` would be null (since you only create it after it's bootstrapped) ;)